client/stylesheets/_toCat.scss

```diff
@@ -3678,6 +3678,22 @@ On click the images will be displayed at normal size to complete the effect
     border: 2px solid #f8504b;
 }
 
+.profileHistoryCardUl {
+    background-color: gainsboro;
```

Comment:
Color `gainsboro` should be written in hexadecimal form as `#dcdcdc`<br>Color literals like `gainsboro` should only be used in variable declarations; they should be referred to via variable everywhere else.<br>Line should be indented 2 spaces, but was indented 4 spaces